pom.xml

```diff
@@ -698,13 +698,6 @@
                                     <exclude>com.google.collections:google-collections</exclude>
                                     <!-- but not the badly numbered ones... -->
                                     <exclude>com.google.guava:guava</exclude>
-                                    <!-- Clashes with com.google.code.findbugs:annotations and having both jars -->
-                                    <!-- as a dependency then clashes with the dependency checker (because it   -->
-                                    <!-- is not very good at handling annotations). Use the annotations jar     -->
-                                    <!-- instead which has all jsr305 annotations and the additional findbugs   -->
-                                    <!-- stuff. -->
-                                    <exclude>com.google.code.findbugs:jsr305</exclude>
-                                    <exclude>net.jcip:jcip-annotations</exclude>
```

Comment:
These are pulled in with the new KB dependencies. With these exclusions in the pom, the build fails for KB 0.18.7.